src/docs/sphinx/developerGuide/Contributing/InstallWin.rst

```diff
@@ -81,13 +81,13 @@ There are two things you may have noticed reading through the ``Dockerfile`` :
 
 .. code:: shell
 
-    PS> $env:VERSION='164-677'
-    PS> $env:IMG='ubuntu18.04-gcc8'
+    PS> $env:VERSION='212-910'
```

Comment:
```suggestion
    PS> $env:VERSION='213-913'
```

according to https://hub.docker.com/layers/geosx/ubuntu20.04-gcc9/213-913/images/sha256-4e20100e6a333a098a635dec3226380ca5f9131501723563090a4dc599582e63?context=explore

Reply:
PR `213` is still not merged, and if/when it is, it's not sure that it will be using built `913`.
I prefer to stick to current TPL tag `212-910` https://github.com/GEOSX/GEOSX/blob/develop/.travis.yml#L10 which is sure to have been working at least once! 🤣

Reply:
Oups my bad, just read the newest ! sorry